pyaedt/generic/configurations.py

```diff
@@ -639,7 +639,7 @@ def set_all_import(self):
 
 class ImportResults(object):
     """Import Results Class.
-    Contains the results of the import operations. Each reusult can be ``True`` or ``False``.
+    Contains the results of the import operations. Each result can be ``True`` or ``False``.
```

Comment:
```suggestion
    Contains the results of the import operations.
    
    Each result can be ``True`` or ``False``.
```
The class description should be a brief sentence fragment with any necessary information following in clear, complete sentences.